src/phonebattery.c

```diff
@@ -0,0 +1,80 @@
+#include <pebble.h>
+#include "configs.h"
+#include "phonebattery.h"
+#include "keys.h"
+#include "text.h"
+
+#if !defined PBL_PLATFORM_APLITE
+
+static bool phonebattery_enabled;
+static int last_update = 0;
+static int phonebattery_interval = 5;
+static int phonebattery_expiration = 30;
+static AppTimer *retry_timer;
+
+static void retry_handler(void *context) {
```

Comment:
indentation on these too :)

Reply:
Is it ok now? Sorry, same as previous chunk - i am bad at c, so i tried to autoformat with my editor.